inst/extdata/SEINRDconfig.yaml

```
@@ -43,7 +43,7 @@ register:
   unique_name: Simple model parameters - External force of infection
   alternate_identifier_type: simple_model_params
   file_type: csv
-  release_date: ${{DATETIME}}
+  release_date: 2022-01-28T12:00
```

Again, I'm not sure why you've removed the `${{DATETIME}}` element:
```suggestion
  release_date: ${{DATETIME}}
```

---

Issue with CLI - ${{DATETIME}} will not work